std_msgs/Header header

# Set of vSGraphs_RoomData.msg
VSGraphsRoomData[] rooms